package.json

```diff
@@ -42,10 +44,14 @@
     "grunt-sass": "^1.1.0",
     "jquery": "1.x",
     "jsdom": "latest",
+    "json2lcov": "^0.1.7",
```

Comment:
You'd have to try it, but I don't _think_ you need `json2lcov` or `mocha-lcov-reporter` any more, as it's all handled by `mochify-istanbul`